File: app/src/main/java/me/askfix/api/askfix/centrifuge/AppConnectHandler.java
package me.askfix.api.askfix.centrifuge;

import android.widget.TextView;
import android.content.Context;

import centrifuge.Client;
import centrifuge.ConnectEvent;
import centrifuge.ConnectHandler;
import me.askfix.api.askfix.MainActivity;
import me.askfix.api.askfix.R;

public class AppConnectHandler implements ConnectHandler {
    private MainActivity context;

    public AppConnectHandler(Context context) {
        this.context = (MainActivity) context;
    }

    @Override
    public void onConnect(Client client, final ConnectEvent event) {
        context.runOnUiThread(() -> {
            TextView tv = context.findViewById(R.id.tvConnectStatus);
            tv.setText("Connected with client ID " + event.getClientID());
        });
    }
}

File: app/src/main/java/me/askfix/api/askfix/centrifuge/AppConnectHandler.kt
package me.askfix.api.askfix.centrifuge

import android.widget.TextView
import android.content.Context
import android.app.Activity
import android.view.View
import centrifuge.Client
import centrifuge.ConnectEvent
import centrifuge.ConnectHandler
import me.askfix.api.askfix.MainActivity
import me.askfix.api.askfix.R

class AppConnectHandler(context: Context) : ConnectHandler {
    private var context: MainActivity = context as MainActivity

    override fun onConnect(client: Client, event: ConnectEvent) {
        context.runOnUiThread {
            val tv = (context as Activity).findViewById<View>(R.id.tvConnectStatus) as TextView
            tv.text = "${context.getString(R.string.connected)} ${event.clientID}"
        }
    }
}